Omikron/Factfinder/Helper/Data.php

```diff
@@ -45,10 +46,21 @@ public function __construct(
 
     /**
      * Public Getter
+     * @param null|int|string $scopeCode
```

Comment:
Why did you rename it to $scopeCode when you pass the store id as parameter? The previous doc name did fit better to the current argument from line 85 Communication.php.